packages/infrastructure/wf_dockerfile_build/main.tf

```diff
@@ -166,7 +166,7 @@ module "image_builder_workflow" {
       cpu    = "25m"
     }
     limits = {
-      memory = "100Mi"
+      memory = "${var.memory_mb}Mi"
```

Comment:
Need to adjust the request as well. I typically do limits at 1.15-1.30x the request.

Reply:
didn't add the multiplier as i didn't see it happening on wf_deploy. i did however add the cpu millicore var as well